src/lib.rs

```diff
@@ -66,7 +65,8 @@
     // TODO: we need this because of rust-typemap.
     // Stop allowing this warning when this PR gets accepted upstream:
     // https://github.com/reem/rust-typemap/pull/44
-    where_clauses_object_safety
+    where_clauses_object_safety,
+    clippy::needless_doctest_main
```

Comment:
This can be avoided by removing `fn main()` from [here](https://github.com/maidsafe/maidsafe_utilities/blob/a5b80bb2890a24749da66c6416a10bcd7bd476f9/src/log_or_panic.rs#L20)

Reply:
I tried that but then one test was failing.

Reply:
Ah it's fine I guess. Can deal with it later.